OpenRA.Game/Scripting/ScriptContext.cs

```diff
@@ -86,8 +86,8 @@ public abstract class ScriptGlobal : ScriptObjectWrapper
 		public ScriptGlobal(ScriptContext context)
 			: base(context)
 		{
-			// The 'this.' resolves the actual (subclass) type
-			var type = this.GetType();
+			// GetType resolves the actual (subclass) type
+			var type = GetType();
```

Comment:
Specifically: IIRC, when I wrote this, the `this.` was needed to make this not return `ScriptGlobal` for everything.